src/FredData.jl

```diff
@@ -67,20 +67,29 @@ mutable struct Fred
     end
 end
 Fred(key::AbstractString) = Fred(key, DEFAULT_API_URL)
-function Fred()
-    key = if KEY_ENV_NAME in keys(ENV)
+
+function fred_key()
```

Comment:
nice idea to separate out this functionality. can you please rename it `load_fred_key` to be more explicit about what the method is doing?